go.mod

```diff
@@ -3,9 +3,10 @@ module github.com/k8stopologyawareschedwg/noderesourcetopology-api
 go 1.15
 
 require (
+	github.com/k8stopologyawareschedwg/noderesourcetopology-api/pkg/apis/topology v0.0.0
```

Comment:
we had github.com/k8stopologyawareschedwg/noderesourcetopology-api  module,
and this patch introduces additional github.com/k8stopologyawareschedwg/noderesourcetopology-api/pkg/apis/topology, right? As dependency of github.com/k8stopologyawareschedwg/noderesourcetopology-api.

Reply:
That was the idea but this is now dropped in the latest version